src/app/helptext/task-calendar/replication/replication-wizard.ts

```diff
@@ -4,19 +4,30 @@ export default {
     step1_label: T('What and Where'),
 
     exist_replication_placeholder: T('Load Previous Replication Task'),
-    exist_replication_tooltip: T(''),
+    exist_replication_tooltip: T('Fill the wizard with the settings from \
+ a saved replication.'),
 
     source_datasets_from_placeholder: T('Source Datasets'),
-    source_datasets_from_tooltip: T(''),
+    source_datasets_from_tooltip: T('Datasets that have snapshots to \
+ replicate. Define the location of the source datasets, then select \
+ datasets for replication. Choosing <i>On a Different System</i> \
+ requires configuring an SSH connection to the remote system. '),
```

Comment:
This is confusing, partly due to the "requires configuring" wording. It's not really clear what this is trying to say.  Possibly that remote datasets can only be chosen from systems with an SSH connection, either existing or added right now.